### DISCUSSION TITLE
Turbo hotkey/button/switch

### DISCUSSION THREAD

**Discussion by WolfRamiO74** [darthcloud/BlueRetro#881] [Ideas]
Yes, I did a search regarding this before posting...

Is there a way to configure a button from gamped bluetooth to activate and deactivate the turbo of a specific button?

For example, assign the "XBOX" button to activate/deactivate the turbo on the R2 of the emulated PS2 dualshock.

Thank you!!!

**Comment by darthcloud** [comment]
No that's not possible, Blueretro don't have any "modifier" type of button function. That may come in the future but likely not this year.
